hazelcast/src/test/java/com/hazelcast/cache/CacheConfigTest.java

```diff
@@ -203,6 +199,19 @@ public void defaultCacheTest() {
     }
 
     @Test
```

Comment:
@bilalyasar 
As the matter of fact NullPointer occurs here:
```
cacheManager.createCache(cacheName, null);
```
and it won't check the second row:
```
assertNotNull(cache);
```
that's why putting ```@Test(expected = NullPointerException.class)``` won't work better than current one, visa versa it's more confusing.

But for others situations using ```@Test ...``` is simpler.